metadata-etl/src/main/resources/jython/CodeSearchExtract.py

```diff
@@ -128,7 +128,11 @@ def run(self):
                 self.logger.info("Search request {}".format(prop_file))
 
             code = result['elements'][0]['docData']['code']
-            code_dict = dict(line.split("=", 1) for line in code.strip().splitlines())
+            try:
+                code_dict = dict(line.split("=", 1) for line in code.strip().splitlines())
+            except:
+                self.logger.error("Exception happens with code {}".format(code))
+
             if "database.name" in code_dict:
                 db['database_name'] = code_dict['database.name']
```

Comment:
Good catch. Modified the exception handling, it will catch both keyError and valueError